staging/src/k8s.io/sample-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -99,10 +100,20 @@ func (c completedConfig) New() (*WardleServer, error) {
 		GenericAPIServer: genericServer,
 	}
 
+	flunderREST, err := flunderstorage.NewREST(Scheme, c.GenericConfig.RESTOptionsGetter)
```

Comment:
@sttts @p0lyn0mial Note to us.  We may want a `MustREST` (or `RESTInPeace`?) function that panics on an error just to make this look prettier.